app/src/main/java/ch/epfl/sdp/artificial_intelligence/PolarPoint.java

```diff
@@ -1,13 +1,13 @@
 package ch.epfl.sdp.artificial_intelligence;
 
 public class PolarPoint extends GenPoint{
-    public PolarPoint(float r, float theta) {
+    public PolarPoint(double r, double theta) {
```

Comment:
Just a small question, why finally we choose double instead of float?

Reply:
We may need this kind of precision because of the conversions between GeoPoint and GenPoint.